### PR TITLE
Add article on hidden BeatSaver features with images

### PR DESCRIPTION
**Pull request by TimothyGillespie**

### CODE REVIEW

**Comment by Pixelguymm** [comment]
Some more:
- Using [`mapper:{name}`](https://beatsaver.com/?q=mapper%3Apixelguy%20bad%20computer) you can filter to maps made exclusively by that mapper.
- Entering a map hash into the search bar will bring you directly to the map corresponding to that hash, similar to `key:`.

Also, you could group these in to sections like "Search", "Playlists", etc.

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (Pixelguymm) [inline src/collections/posts/hidden-features-of-beatsaver.md:29]: For these keyboard keys, you *could* use the `<kbd>` tag, and give it a nice style to have it stand out from code blocks.
The [JetBrains docs](https://www.jetbrains.com/help/phpstorm/wrapping-a-tag-example-of-applying-surround-live-templates.html) do this quite nicely.

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:48]: Lines 39-45 are a bit confusing and possibly misleading unless I'm not following it correctly. Pressing Shift and Alt together will filter out all others, but also allows for maps that have no tags to display so you won't find your techy fitbeat maps. You can Shift select Tech and Fitbeat but it'll still be "either/or" instead of "and". As far as I know, there is no way to do the "and" query (I think this would've been part of the PR we previously discussed for BeatSaver).

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:60]: For example, if I wanted to do this to my bookmarks, I'd type in: `https://beatsaver.com/playlists/97886/add` (your URL will have a different playlist ID in the URL so this link will not work for you).

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:62]: You can also add `/edit` to the URL, but this will not do a lot for you since the API will not allow you to save edits to the bookmarks playlist.

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:88]: When you try to upload a map that has already been uploaded, it will not allow you to do so and will tell you that the map has already been uploaded. This is checked via the map hash. The upload will also generally verify that your map file is valid and will work in-game.

**Comment by TimothyGillespie** [comment]
Thank you! I've integrated that. I though the mapper: one was a fluke, but apparently it's real :smile: not sure why I thought otherwise. And I had no idea about the hash. About the grouping: I didn't quite understand or see the vision. All I could imagine with that would come out odd.

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:94]: There is, however, no notification that you review has been curated.

File: src/collections/posts/hidden-features-of-beatsaver.md
Comment (TheCzar1994) [inline src/collections/posts/hidden-features-of-beatsaver.md:100]: If you ever wondered where all the bot maps are that people talk about, then you may not be aware of the fact that BeatSaver, by default, hides bot maps.

**Review by TheCzar1994 — COMMENTED** [review]
All just pretty minor grammatical suggestions except on Line 45.

**Review by TheCzar1994 — APPROVED** [review]
This looks good to me - thank you for writing an article!
Looks like we were waiting on a review from Pixel so I'll let him merge when he's good with it too.

**Pull request merged**
